### PR TITLE
Feature/4434 example i18n instance merge

### PR DESCRIPTION
**Pull request by ektaghag-eaton**
Fixes 4434 .

#### Changes proposed in this Pull Request:

- custom screen translations in workflow http://localhost:3000/self-registration
- Auth workflow translations should work http://localhost:3000/forgot-password
- Registration workflow should work
- Pages after login should translate http://localhost:3000/homepage

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

- custom screen translations in workflow http://localhost:3000/self-registration
- Auth workflow translations should work http://localhost:3000/forgot-password
- Registration workflow should work
- Pages after login should translate http://localhost:3000/homepage

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: login-workflow/example3/src/screens/CustomScreen.tsx
[the remark this answers is not in the document]
Comment (ektaghag-eaton) [reply login-workflow/example3/src/screens/CustomScreen.tsx:1]: This is only for testing purposes. Will remove it once it is tested correctly .

**Review by surajeaton — COMMENTED** [review]
Getting this error while running example 3:
Screenshot 2023-07-24 at 7 04 57 PM

**Comment by ektaghag-eaton** [comment]
> Getting this error while running example 3: Screenshot 2023-07-24 at 7 04 57 PM

 Ffolder name has been renamed. Can you try after deleting dist and nodemodules?

**Review by surajeaton — DISMISSED** [review]
Looks good. @ektaghag-eaton Please remove the code added for testing.

**Review by surajeaton — COMMENTED** [review]
Please resolve the conflicts.

**Review by surajeaton — APPROVED** [review]
Looks good. Please remove the CustomScreen before merging.

**Pull request merged**